Android/Shared/.gitignore

```diff
@@ -1 +1,19 @@
-/ITMSamples.properties
+*.iml
```

Comment:
You lost ITMSamples.properties, which is supposed to be in this file.

Reply:
Fixed